### ISSUE TITLE
find command not found in ubi-minimal:8.4-210 container

### ISSUE THREAD

**Issue by geoff-carr-bzy** [jfrog/charts#1548]
It seems as if the `find` command is missing from the `ubi-minimal:8.4-210` `initContainerImage` now used in Xray since the latest chart version.

The  `copy-custom-certificates` container fails with the following error: -

```
Copy custom certificates to /var/opt/jfrog/xray/etc/security/keys/trusted
sh: find: command not found
sh: find: command not found
```

The following command is what is trying to execute: -

```
- command:
        - /bin/sh
        - -c
        - |
          echo "Copy system.yaml to /var/opt/jfrog/xray/etc"; mkdir -p /var/opt/jfrog/xray/etc; cp -fv /tmp/etc/system.yaml /var/opt/jfrog/xray/etc/system.yaml; echo "Remove /var/opt/jfrog/xray/lost+found folder if exists"; rm -rfv /var/opt/jfrog/xray/lost+found; echo "Copy joinKey to /var/opt/jfrog/xray/etc/security"; mkdir -p /var/opt/jfrog/xray/etc/security; echo ${XRAY_JOIN_KEY} > /var/opt/jfrog/xray/etc/security/join.key; echo "Copy masterKey to /var/opt/jfrog/xray/etc/security"; mkdir -p /var/opt/jfrog/xray/etc/security; echo ${XRAY_MASTER_KEY} > /var/opt/jfrog/xray/etc/security/master.key;
```

The same command is used in the Insight deployment, however is uses a different container image which is `ubi-micro/base/insight:1`

**Comment by amithins** [comment]
This has been fixed as part of https://github.com/jfrog/charts/pull/1547.

**Comment by geoff-carr-bzy** [comment]
Can this be reopened please, still seem to be having the same issue with the latest version of the `jfrog-platform` chart: -

```
Copy custom certificates to /var/opt/jfrog/xray/etc/security/keys/trusted
bash: find: command not found
bash: find: command not found
```

**Comment by chukka** [comment]
Please raise a new Issue , also specify chart version of platform chart , today we released 10.0.4 , please verify on that chart too.
